File: src/libs/ThirdParty.VBSuppliers/Properties/ExpediaRapid/SerializableClasses/OccupancyRateTotal.vb
Imports Newtonsoft.Json
Namespace ExpediaRapid.SerializableClasses
    Public Class OccupancyRateTotal

        <JsonProperty("billable_currency")>
        Public Property TotalInBillableCurrency As OccupancyRateAmount

        <JsonProperty("request_currency")>
        Public Property TotalInRequestCurrency As OccupancyRateAmount

    End Class
End Namespace

File: src/libs/iVectorOne.Suppliers/ExpediaRapid/SerializableClasses/OccupancyRateTotal.cs
namespace ThirdParty.CSSuppliers.ExpediaRapid.SerializableClasses
{
    using Newtonsoft.Json;

    public class OccupancyRateTotal
    {

        [JsonProperty("billable_currency")]
        public OccupancyRateAmount TotalInBillableCurrency { get; set; }

        [JsonProperty("request_currency")]
        public OccupancyRateAmount TotalInRequestCurrency { get; set; }

    }
}